ditados.json

```diff
@@ -15,6 +15,14 @@
       {
          "content": "De grão em grão, a galinha enche o papo",
          "meaning": "Quando economizamos pouco, mas constantemente, conseguimos alcançar aquilo que desejamos, mesmo se o objetivo final pareça ser impossível"
+      },
+      {
+         "content": "Devagar se vai longe",
+         "meaning": "Apresenta que pessoas que fazem atividades com calma, cada uma a seu tempo, conseguem ter mais sucesso do que as que realizam seus afazeres apressados."
+      },
+      {
+         "content": "Cão que ladra não morde",
+         "meaning": "mostra que algumas pessoas ameaçam com palavras, mas, na verdade, não fazem nada, por isso não é necessário temê-las."
```

Comment:
Please capitalize the 'M' in 'mostra'. Thanks!

Reply:
No problem! I'll edit